std/file.d

```diff
@@ -2287,21 +2287,21 @@ if (isConvertibleToString!R)
 // Same as mkdir but ignores "already exists" errors.
 // Returns: "true" if the directory was created,
 //   "false" if it already existed.
-private bool ensureDirExists(in char[] pathname)
+private bool ensureDirExists(in char[] pathname) @safe
 {
     import std.exception : enforce;
 
     version(Windows)
     {
-        if (CreateDirectoryW(pathname.tempCStringW(), null))
+        if (() @trusted { return CreateDirectoryW(pathname.tempCStringW(), null); }())
```

Comment:
`mkdir` does `auto pathz = pathname.tempCString!FSChar();`, same could be done here